From cb047d4ffa6c3c729961cbfeebb89ed47cc417d8 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Wed, 4 Apr 2018 16:36:45 +0800
Subject: [PATCH] Revert "arm64: dts: rockchip: rk3308: Assign emmc, sdio,
 sdmmc clocks to DIV50"

This reverts commit 699150fe82eca7a1ac8c07412d207938eab1cb0c.

Change-Id: I83f6476540d71e0eba45e27b29f585bb789ee36f
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308.dtsi | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308.dtsi b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
index 9c456f9110ac..e5d058a6a6af 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
@@ -793,8 +793,6 @@
 		clocks = <&cru HCLK_SDMMC>, <&cru SCLK_SDMMC>,
 			 <&cru SCLK_SDMMC_DRV>, <&cru SCLK_SDMMC_SAMPLE>;
 		clock-names = "biu", "ciu", "ciu-drv", "ciu-sample";
-		assigned-clocks = <&cru SCLK_SDMMC>;
-		assigned-clock-parents = <&cru SCLK_SDMMC_DIV50>;
 		fifo-depth = <0x100>;
 		interrupts = <GIC_SPI 76 IRQ_TYPE_LEVEL_HIGH>;
 		pinctrl-names = "default";
@@ -810,8 +808,6 @@
 		clocks = <&cru HCLK_EMMC>, <&cru SCLK_EMMC>,
 			 <&cru SCLK_EMMC_DRV>, <&cru SCLK_EMMC_SAMPLE>;
 		clock-names = "biu", "ciu", "ciu-drv", "ciu-sample";
-		assigned-clocks = <&cru SCLK_EMMC>;
-		assigned-clock-parents = <&cru SCLK_EMMC_DIV50>;
 		fifo-depth = <0x100>;
 		interrupts = <GIC_SPI 77 IRQ_TYPE_LEVEL_HIGH>;
 		status = "disabled";
@@ -825,8 +821,6 @@
 		clocks = <&cru HCLK_SDIO>, <&cru SCLK_SDIO>,
 			 <&cru SCLK_SDIO_DRV>, <&cru SCLK_SDIO_SAMPLE>;
 		clock-names = "biu", "ciu", "ciu-drv", "ciu-sample";
-		assigned-clocks = <&cru SCLK_SDIO>;
-		assigned-clock-parents = <&cru SCLK_SDIO_DIV50>;
 		fifo-depth = <0x100>;
 		interrupts = <GIC_SPI 78 IRQ_TYPE_LEVEL_HIGH>;
 		pinctrl-names = "default";
-- 
2.35.3

